test/graphql/queries/models_query_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'test_helper'
+
+class Queries::ModelsQueryTest < ActiveSupport::TestCase
+  test 'fetching models attributes and relations' do
+    model = FactoryBot.create(:model)
+
+    query = <<-GRAPHQL
+      query modelsQuery {
```

Comment:
I assume the `pageInfo`, `edges`, `cursor` and `node` stuff is related to pagination, right? This query is currently failing for me.

```
2019-02-07T15:07:38 [F|app|fc79434a] RuntimeError (        Failed to implement BaseConnection.pageInfo, tried:
 | 
 |         - `#<Class:0x007fe30e4f3de8>#page_info`, which did not exist
 |         - `Model::ActiveRecord_Relation#page_info`, which did not exist
 |         - Looking up hash key `:page_info` or `"page_info"` on `#<Model::ActiveRecord_Relation:0x007fe30a8dd2f0>`, but it wasn't a Hash
 | 
 |         To implement this field, define one of the methods above (and check for typos)
 | ):
2019-02-07T15:07:38 [F|app|fc79434a]   
2019-02-07T15:07:38 [F|app|fc79434a] app/controllers/api/graphql_controller.rb:47:in `execute_single_graphql_query'
 | app/controllers/api/graphql_controller.rb:18:in `execute'
 | app/controllers/concerns/foreman/controller/timezone.rb:10:in `set_timezone'
 | app/models/concerns/foreman/thread_session.rb:32:in `clear_thread'
 | lib/foreman/middleware/telemetry.rb:10:in `call'
 | lib/foreman/middleware/catch_json_parse_errors.rb:9:in `call'
 | lib/foreman/middleware/logging_context_session.rb:22:in `call'
 | lib/foreman/middleware/logging_context_request.rb:11:in `call'
```

Reply:
Yes, they are used for pagination. I guess this needs some code that will be added in a future PR.

Note to self: `implements GraphQL::Relay::Node.interface` might help.